### PR TITLE
Add biteau catalog

### PR DESCRIPTION
**Pull request by pdeiml**
Es sind alle Daten in 
/input/data/YEAR/PAPER_IDENTIFIER/tev-SOURCE_ID-sed.ecsv
gespeichert.
Ausserdem alle fehlenden Referenzen in
/input/source/tev-SOURCE_ID.yaml
eingetragen.
Das eigentlich entscheidende Skript ist
gamma-cat/other_data_collections/2015ApJ...812...60B/biteau.py

### CODE REVIEW

**Comment by pdeiml** [comment]
Ausserdem habe ich dein biteau.py in bitrau_old.py umbenannt und ich musste den BiteauWiliams Katalog leicht verändern. Deswegen habe ich auch davon eine Backup Kopie gemacht: BiteauWilliams..._old.ecsv

File: input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+0.980000218484 0.0 1.30200144389e-10 9.83449608154e-12 9.83449608154e-12
+1.21199873196 0.0 6.27303291071e-11 8.98552643031e-12 8.98552643031e-12
+1.4980008244 0.0 3.3099930624e-11 8.6461757958e-12 8.6461757958e-12
+1.8510007513 0.0 1.66099892575e-11 8.22630908124e-12 8.22630908124e-12
+2.28800026098 0.0 1.12399967018e-11 9.06691573153e-12 9.06691573153e-12
+2.82799851617 0.0 6.15800716533e-12 9.1012436633e-12 9.1012436633e-12
+3.49599973938 0.0 3.34700164694e-12 9.07727690936e-12 9.07727690936e-12
+4.32098696108 0.0 1.16000723495e-12 7.52632406144e-12 7.52632406144e-12
+5.34200062268 0.0 5.22999528178e-13 6.76610843597e-12 6.76610843597e-12
+0.70099983322 0.0 3.87099976121e-10 2.06290005741e-11 2.06290005741e-11
```
Comment (cdeil) [inline input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv]: There's a problem here with `input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv`.
It contains ~ 100 rows. There must be a scripting issue, there aren't that many flux points for any source.

**Comment by cdeil** [comment]
More comments:

- Please remove the `_old` version of the Python script and Biteau ECSV file. That's what version control is for. We don't need to store old versions of scripts, this is just confusing.

- The formatting of the Python script hurts my eyes. Can you please run [autopep8](https://pypi.python.org/pypi/autopep8) on the script, or directly use a good Python code editor (I use Pycharm) where you can just selection "format source code"?

- Reading the script it's hard for me to see if the factors you put to convert fluxes are correct or not. Can you rewrite those lines to directly use Astropy quantities?
See http://astropy.readthedocs.io/en/latest/units/index.html

**Comment by pdeiml** [comment]
Hi,

I am not sure what you exactly mean with your remark about the e_ref_err.
The file input/data/2004/2004Natur.432...75A/tev-000096-sed.ecsv was not added by my script. 
It is already in the master-branch.

I agree that in the Biteau catalog it seems that e_ref_err is always 0. 
When I wrote the python-script I only looked at the catalog and added all relevant columns to the produced sed-ecsv-files.

Do you want me to remove all e_ref_err columns read from the Biteau catalog because they are 0 or do you want to remove the e_ref_err columns in general?

File: input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv (hunk 1 above)
Comment (pdeiml) [inline input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv]: What do you mean? 
input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv contains 22 lines, doesn't it?

**Comment by cdeil** [comment]
@pdeiml - I think we talked about all remaining issues to wrap this up, no?
Let me know if you have any questions or if I should do a final review or merge.

**Pull request closed without merging**